npu.user.js

```diff
@@ -1049,9 +1062,12 @@ var npu = {
 					
 					$("tbody tr[id*=tr__]", table).each(function() {
 						var row = $(this);
+						var courseCode = $("td:nth-child(3)", row).clone().children().remove().end().text();

```

Comment:
Simply calling `text()` in the previous version appended the tooltip "Már jelentkezett!" to the end of the text, causing an invalid subject code to be recorded. Solution taken from here: http://stackoverflow.com/questions/3442394/using-text-to-retrieve-only-text-not-nested-in-child-tags